compliance_checker/cf/cf.py

```diff
@@ -132,7 +133,7 @@ def __init__(self):
         self._metadata_vars  = defaultdict(list)
         self._boundary_vars  = defaultdict(dict)
 
-        self._std_names      = StandardNameTable('cf-standard-name-table.xml')
+        self._std_names      = StandardNameTable('data/cf-standard-name-table.xml')
```

Comment:
Could you use `pkg_resources` here? This path is only valid if you're running this from the project root, anywhere else, this path is very likely to be invalid.